scripts/kibana.js

```diff
@@ -6,4 +6,10 @@
  * Side Public License, v 1.
  */
 
+// Kibana already handles these events internally for graceful shutdowns.
+// `src/cli/dev` is spawning Kibana in a child process to apply the preserve-symlinks options.
+// We need to catch these events here to avoid killing the parent process before Kibana (the child) is fully finished.
+process.on('SIGINT', function () {});
+process.on('SIGTERM', function () {});
```

Comment:
@spalger does that look alright to you?

Reply:
I moved it to the place where we are really spawning the children in the commit [`644d0e1` (#97223)](https://github.com/elastic/kibana/pull/97223/commits/644d0e11b73dd8678e408975c13244d47f52379d)